recipes/multirnafold/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "MultiRNAFold" %}
+{% set version = "2.1" %}
+{% set sha256 = "cb5462e03c70da24caaa9706f493fa329b1e15f697fbffd5bf460cd873243306" %}
+
+package:
+  name: {{ name | lower }}
+  version: {{ version }}
+
+build:
+  number: 0
+  skip: True  # [osx]
```

Comment:
why do you skip OSX builds? maybe add a comment

Reply:
I managed to fix the OSX build with a patch